### PR TITLE
Update install.rst

### PR DESCRIPTION
**Pull request by agrimwood**
Introductory text has been altered to clarify "the package". Minor spelling mistake corrected further down.
#322

# Description

Introductory text has been altered to clarify "the package". Minor spelling mistake corrected further down.

Fixes # 322

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not
      work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so
we can reproduce. Please also list any relevant details for your test configuration If
there are no tests written, flag with label so we can help generate some tests before
merge.

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Firmware version:
- Hardware:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by zacbaum** [comment]
@agrimwood, if we can hold off on this until the pypi package PR is merged - this will require a few changes!

See: https://github.com/DeepRegNet/DeepReg/pull/334/files

**Review by YipengHu — COMMENTED** [review]
@agrimwood can you liase with @zacbaum to also update per the pypi

**Comment by zacbaum** [comment]
@agrimwood master should now contain updates to install.rst - if you can update from master and then make changes there, that will work now! Thanks 😄

**Review by DeepRegNet — APPROVED** [review]

**Pull request merged**
